package.json

```diff
@@ -43,7 +43,6 @@
   },
   "devDependencies": {
     "@babel/types": "~7.21.4",
-    "@sentry-internal/eslint-config-sdk": "^7.48.0",
```

Comment:
For a sec I was like, who approved us not using our central config?

And then I realized it was me 😅 https://github.com/getsentry/sentry-wizard/pull/252

throwback to when I added it though https://github.com/getsentry/sentry-wizard/pull/72